metis-validation/metis-validation-service/src/test/java/TestValidationExecution.java

```diff
@@ -105,8 +106,31 @@ public void testSingleValidationFailure() throws Exception {
         String fileToValidate = IOUtils.toString(new FileInputStream("src/test/resources/Item_35834473_wrong.xml"));
         ValidationResult result = validationExecutionService.singleValidation(EDM_INTERNAL, "EDM-INTERNAL.xsd", "schematron/schematron-internal.xsl", fileToValidate);
         Assert.assertFalse(result.isSuccess());
-        Assert.assertNotNull(result.getRecordId());
-        Assert.assertNotNull(result.getMessage());
+        Assert.assertEquals("Missing record identifier for EDM record", result.getRecordId());
+        Assert.assertNull(result.getNodeId());
+        Assert.assertEquals("cvc-complex-type.2.4.a: Invalid content was found starting with element 'edm:WebResource'. One of '{\"http://www.europeana.eu/schemas/edm/\":ProvidedCHO}' is expected.", result.getMessage());
+    }
+
+    @Test
+    public void shouldFailOnSchematronValidation() throws Exception {
+
+        String fileToValidate = IOUtils.toString(new FileInputStream("src/test/resources/Item_schematron_invalid.xml"));
```

Comment:
Change all the IOUtils.toString methods to something like e.g.
IOUtils.toString(new FileInputStream("src/test/resources/Item_schematron_invalid2.xml"), StandardCharsets.UTF_8);
And if StandardCharsets.UTF_8 doesn't work then use "UTF-8" as a second parameter.